pandas/_libs/include/pandas/vendored/ujson/lib/ultrajson.h

```diff
@@ -137,6 +137,9 @@ typedef int64_t JSLONG;
 #error "Endianness not supported"
 #endif
 
+// PANDAS_FORMAT is *not* in the original file this is vendored from
```

Comment:
Any reason to move this? Unless it is used elsewhere should just stay in the module

Reply:
i think i did this for a branch where i was trying to reimplement bits of the objToJSON.c in cython.  will revert this bit.